PokeAlarm/Events/MonEvent.py

```diff
@@ -159,7 +160,16 @@ def generate_dts(self, locale, timezone, units):
             # Spawn Data
             'spawn_start': self.spawn_start,
             'spawn_end': self.spawn_end,
-            'spawn_verified': self.spawn_verified,
+            'spawn_verified':
```

Comment:
We should change this to `verified` or change the rest to include `spawn_` in front of them for consistency